js/build.js

```diff
@@ -159,61 +151,46 @@ $('[data-login-id]').each(function() {
   }
 
   function init() {
-    Fliplet.Security.Storage.init()
-      .then(function(){
-        return Fliplet.Security.Storage.create(_this.pvName, dataStructure)
-      })
-      .then(function(data){
-        _this.loginPV = data || {};
-
-        if (!data || !_this.loginPV) {
-          showStart();
-          return;
-        }
-
-        if (!Fliplet.Navigator.isOnline()
-          && _this.loginPV.auth_token
-          && !Fliplet.Env.get('disableSecurity')) {
-          Fliplet.Navigate.to(_this.data.action);
-          return;
-        }
-
-        if (_this.loginPV.auth_token === '') {
-          showStart();
-          return;
-        }
+    _this.loginPV = {};
+    Fliplet.User.getCachedSession()
+      .then(function(session) {
+        if (session && session.accounts && session.accounts.flipletLogin) {
+          _this.loginPV = session.accounts.flipletLogin[0];
 
-        validateWeb()
-          .then(function() {
-            return validateAppAccess();
-          })
-          .then(function() {
-            if (Fliplet.Env.get('disableSecurity')) {
-              console.warn('Fliplet Login component tried to navigate to a page, but security is disabled.');
+          if (!Fliplet.Navigator.isOnline() && !Fliplet.Env.get("disableSecurity")) {
+            Fliplet.Navigate.to(_this.data.action);
+            return;
+          }
+          
+          validateWeb()
+            .then(function() {
+              return validateAppAccess();
```

Comment:
@seromenho Should you not be removing this too if you're removing the check for app access against the user?

Reply:
Totally right. Was already done. but with the loss of code and then recovering it I missed that. 
Thanks!! 
Done.